SubPub/Tests/SubPubTests/TestSubscriber.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  TestSubscriber.swift
+//  
+//
+//  Created by Chad on 8/27/20.
+//
+
+import Foundation
+import SubPub
+
+class TestSubscriber: NSObject {
+    public enum TestState {
+        case error(Error)
+        case loading
+        case loaded([TestData])
+    }
+    public var state: TestState = .loading
+    private let manager: TestManager
+    
+    // MARK: - Init
+
+    init(manager: TestManager) {
+        self.manager = manager
+        super.init()
+        manager.publisher.subscribe(self)
+    }
+
+    @objc func refreshData() {
```

Comment:
Does this need to be `@objc`?

Reply:
good catch; actually the func isn't being used at all right now, so the whole thing can go.